src/applications/appeals/shared/components/IssueCard.jsx

```diff
@@ -190,8 +193,13 @@ export const IssueCard = ({
               onChange={handlers.onChange}
               aria-describedby={`issue-${index}-description`}
               aria-labelledby={`issue-${index}-title`}
+              data-dd-action-name="Issue Name"
             />
-            <label className="schemaform-label" htmlFor={elementId}>
+            <label
+              className="schemaform-label"
+              htmlFor={elementId}
+              data-dd-action-name="Contestable Issue Name"
```

Comment:
Is there a difference between `Issue Name` and `Contestable Issue Name`?

Reply:
We're using different action names (and capitalizations) so that when I click on it in staging, we know where the event text is coming from